frontend/src/Settings/Tags/Details/TagDetailsModalContent.js

```diff
@@ -156,6 +157,21 @@ function TagDetailsModalContent(props) {
               }
             </FieldSet>
         }
+
+        {
+          !!indexers.length &&
```

Comment:
Please convert this to a ternary (we're converting things as we go).

Reply:
Do you have an example of what this should look like? I've not done much frontend stuff before so this was mostly dev by copy/paste. If you can provide an example I'll have a look at updating the other ones as well.

Reply:
```
indexers ?
  <FieldSet legend="Indexers">
  ...
  </FieldSet> : 
  null
```